src/content/protocols/pendle.md

```diff
@@ -0,0 +1,425 @@
+
+## Router System and Action Modules
+
+The `PendleRouterV4` functions as the protocol's central interface hub, directing user interactions to specialized execution modules. This architecture employs a selector-based routing mechanism where each function call is mapped to one of seven dedicated action modules, each responsible for specific protocol operations.
+
+The primary modules include `ActionAddRemoveLiqV3` for liquidity management in Pendle pools, `ActionSwapPTV3` for Principal Token exchange operations, and `ActionSwapYTV3` for Yield Token transactions. Supporting modules such as `ActionMiscV3` handle yield redemption, token conversions, and position exits, while `ActionSimple` provides core utility functions. Advanced functionalities are enabled through `ActionAggregator` for batched operations and `ActionLimitOrder` for conditional trades.
+
+Unlike conventional Transparent proxies, the routing infrastructure of `PendleRouterV4` relies on a diamond proxy with `ActionStorageV4` to manage function selector mappings. The owner of the proxy is set to the governance multisig. By updating the different contracts which receive calls via the `PendleRouterV4` new endpoints can be established.
+
+## Limit Order System
+
+The protocol implemented the `PendleLimitRouter` contract which enables limit orders of PT and YT. 
+
+Users can create orders that execute only when specific price conditions are met and can cancel their orders individually or in batches. The system uses EIP-712 signatures for order validation and employs logarithmic calculations for fee determination. These fee parameters are configured via `setLnFeeRateRoots` by either `Governance` or the `hardwareDeployer`, while collected fees are directed to `Treasury1`.
+
+## PENDLE incentives and Fees
+
+### Fee Collection and Distribution Path
+
+The protocol implements a multi-stage fee distribution architecture with distinct flows for different market types. Fee revenue generated through market operations flows into two separate Treasury multisig accounts: `Treasury1` (2/6 multisig) processes fees from pools created before October 8, 2024, while `Treasury2` (2/5 multisig) handles those from newer pools.
+
+From these Treasury accounts, tokens follow a standardized conversion path. The `hardwareDeployer` EOA serves as an intermediary conversion agent, receiving tokens from both Treasury multisigs. This EOA transforms various token types into ETH through the `PendleRouterV4`, then directs this ETH to the `PendleFeeDistributorV2Proxy` for final distribution.
```

Comment:
This is an EOA, I would emphasize that nothing guarrentees that it is going to distribute the fees